### PR TITLE
Update bootstrap-table-nl-NL.js

### PR DESCRIPTION
**Pull request by wenzhixin**
Up to date to 1.9.1.

### CODE REVIEW

File: dist/locale/bootstrap-table-nl-NL.js [hunk 1]
```diff
@@ -13,15 +13,30 @@
...
         },
         formatShowingRows: function (pageFrom, pageTo, totalRows) {
-            return 'Toon ' + pageFrom + ' tot ' + pageTo + ' van ' + totalRows + ' records';
+            return 'Toon ' + pageFrom + ' tot ' + pageTo + ' van ' + totalRows + ' record' + ((totalRows > 1) ? "s" : "");
         },
-        formatSearch: function () {
+        formatDetailPagination: function (totalRows) {
+            return 'Toon ' + totalRows + ' record' + ((totalRows > 1) ? "s" : "");
+        },
+		formatSearch: function () {
```
Comment (wenzhixin) [inline dist/locale/bootstrap-table-nl-NL.js]: need to update `tab` to `4 spaces`.

File: dist/locale/bootstrap-table-nl-NL.js [hunk 2]
```diff
@@ -13,15 +13,30 @@
...
+        formatRefresh: function () {
+           return 'Vernieuwen';
+        },
+        formatToggle: function () {
+          return 'Omschakelen';
+        },
+        formatColumns: function () {
+          return 'Kolommen';
+        },
+		formatAllRows: function () {
```
Comment (wenzhixin) [inline dist/locale/bootstrap-table-nl-NL.js]: need to update `tab` to `4 spaces`.

File: dist/locale/bootstrap-table-nl-NL.js [hunk 3]
```diff
@@ -13,15 +13,30 @@
...
+          return 'Alle';
         }
-    };
+	};
```
Comment (wenzhixin) [inline dist/locale/bootstrap-table-nl-NL.js]: need to update `tab` to `4 spaces`.

**Comment by BertHankes** [comment]
Done

**Comment by wenzhixin** [comment]
forget to `git push`?

**Comment by BertHankes** [comment]
Try to push, but can't find a way to push it to master on github.com. I just can't find a way to 'push' Sorry.
Maybe you can point out to me how to do that.

**Comment by wenzhixin** [comment]
You have submitted two commits.

**Pull request merged**